storage_service/locations/models/pipeline_local.py

```diff
@@ -136,3 +136,17 @@ def move_from_storage_service(self, source_path, destination_path, package=None)
 
         # Move file
         return self.space.move_rsync(source_path, destination_path, assume_rsync_daemon=self.assume_rsync_daemon, rsync_password=self.rsync_password)
+
+    def isfile(self, path_):
```

Comment:
Is `path` a reserved keyword? I think you could remove the trailing slash, at least it's being used like that in other places of this module.